.gitignore

```diff
@@ -0,0 +1,306 @@
+# Created by https://www.toptal.com/developers/gitignore/api/emacs,visualstudiocode,vim,python,jupyternotebooks,windows,linux,macos,certificates
+# Edit at https://www.toptal.com/developers/gitignore?templates=emacs,visualstudiocode,vim,python,jupyternotebooks,windows,linux,macos,certificates
```

Comment:
Notice that some of these templates we won't need here (`jupyter`, `certificates`) :grimacing: 

```suggestion
# Edit at https://www.toptal.com/developers/gitignore?templates=emacs,visualstudiocode,vim,python,windows,linux,macos
```

To update this template, go to the link above then click on `create` and update the `.gitignore` to that one :smile:

Reply:
Done.